components/team-members/TeamMemberCard.tsx

```diff
@@ -1,11 +1,16 @@
 import { Card, CardActionArea, CardContent } from '@mui/material'
+import Router from 'next/router'
 import React from 'react'
 
 function TeamMemberCard({ teamMember }) {
+  function goToTeamMemberProfile() {
+    Router.push('/team-members/profile/' + teamMember.mail)
+  }
+
   return (
-    <CardActionArea>
+    <CardActionArea onClick={goToTeamMemberProfile}>
```

Comment:
can't we use a <Link> or something similar so that we don't need to create a function?